src/s3fs_util.cpp

```diff
@@ -814,7 +814,7 @@ mode_t get_mode(headers_t& meta, const char* path, bool checkdir, bool forcedir)
             if(string::npos != pos){
               strConType = strConType.substr(0, pos);
             }
-            if(strConType == "application/x-directory"){
+            if(strConType == "application/x-directory" || strConType == "httpd/unix-directory"){
```

Comment:
Could you add a comment explaining what this constant means?

Reply:
Sorry for answering late, have been busy lately.
The comment is: "Nextcloud stores Directory objects with this mime type when mounting a Bucket as external Storage"
I have created a new pull for introducing the Comment

Reply:
Sorry for answering late, have been busy lately.
The comment is: "Nextcloud stores Directory objects with this mime type when mounting a Bucket as external Storage"